src/components/DateTimeField/index.jsx

```diff
@@ -114,16 +120,25 @@ DateTimeField.propTypes = {
     value: PropTypes.string,
     name: PropTypes.string,
     onChange: PropTypes.func.isRequired,
+    placeholder: PropTypes.string,
```

Comment:
This input prop here is coming from the Redux form Field component. From what I can tell, we aren't doing anything with this and you should only need it listed below with the regular props.